### PR TITLE
Complete Arabic translation for OJS

### PR DESCRIPTION
**Pull request by vormia**
Contain all the files needed for Arabic language pack except those of the 'pkp-lib'. These were added separately in another pull request to 'pkp-lib dev 2_4'.

### CODE REVIEW

File: registry/locales.xml [hunk 1]
```diff
@@ -49,4 +49,5 @@
 	<locale key="vi_VN" complete="false" name="Tiếng Việt" iso639-2b="vie" iso639-3="vie" />
 	<locale key="zh_CN" complete="true" name="简体中文" iso639-2b="chi" iso639-3="zho" />
 	<locale key="zh_TW" complete="false" name="繁體中文" iso639-2b="chi" iso639-3="zho" />
+    	<locale key="ar_IQ" complete="true" name="العربية" iso639-2b="ara" iso639-3="ara" stylesheet="../lib/pkp/styles/rtl.css"/>
```
Comment (mtub) [inline registry/locales.xml]: should be:
 ... stylesheet="/lib/pkp/styles/rtl.css" />

Comment (vormia) [inline registry/locales.xml]: @mtub; 
Dots were removed as you noted.

Comment (asmecher) [inline registry/locales.xml]: Marco, are you sure about that leading slash? It looks wrong to me.

Comment (mtub) [inline registry/locales.xml]: Sorry for that, you're right. It should be "lib/pkp/styles/rtl.css", without the leading slash. @vormia , do you mind updating again?

**Comment by mtub** [comment]
@vormia , do you still need the four extra CSS files added to the styles directory? I recommend removing them. The rtl.css file from pkp-lib has a universal rule for right-to-left direction.

**Comment by vormia** [comment]
@mtub;
No, sorry, I forgot them.... they have been removed from the pull request.

**Comment by vormia** [comment]
Any thing else I can do?

**Comment by mtub** [comment]
@vormia, everything looks good now. I will merge your commits. 

Again, thanks for contributing this translation, your help is really appreciated.

**Comment by vormia** [comment]
Thanks all, it was pleasure collaborating with you.

**Comment by asmecher** [comment]
@vormia, many thanks for contributing!

**Pull request merged**